src/components/Text/index.js

```diff
@@ -3,58 +3,73 @@ import PropTypes from 'prop-types';
 import styles from './Text.module.scss';
 
 const Text = props => {
-  const { className, type, src, link, color, weight, id, noUnderline } = props;
+  const {
+    className,
+    typeScale,
+    lineHeight,
+    src,
+    link,
+    inputLabel,
+    color,
+    weight,
+    alignment,
+    style,
+    id
+  } = props;
 
   let addedClasses = '';
   if (link) {
-    addedClasses += styles.Link;
-    if (noUnderline) {
-      addedClasses += ` ${styles.noUnderline}`;
-    }
+    addedClasses += ` ${styles.Link}`;
 
     return (
-      <span
-        className={`text ${className} ${styles[type]} ${addedClasses}`}
+      <a
+        className={`text ${className} ${styles[typeScale]} ${
+          styles[lineHeight]
+        } ${styles[color]} ${styles[alignment]} ${
+          styles[style]
+        } ${addedClasses}`}
         id={id}
+        href={src}
+        target="_blank"
       >
-        <a
-          className={`${styles[color]} ${addedClasses}`}
-          href={src}
-          target="_blank"
-        >
-          {props.children}
-        </a>
-      </span>
+        {props.children}
+      </a>
+    );
+  }
+
+  if (inputLabel) {
+    addedClasses += `${styles.inputLabel}`;
+
+    return (
```

Comment:
@corwinharrell why not make inputLabel a typeScale?

Reply:
The goal here is really just to restrict how inputLabels appear. It ultimately uses the "Small" typeScale size, but there are other properties that I want to apply to all inputLabels for which I don't want to allow change. (Like bottom padding for when they appear above an input etc...)

Reply:
ahhh I see. Ok - lets keep it this way for now. If we find we need to add a few more variations, then we should revisit.

Reply:
👍